### DISCUSSION TITLE
Terminal Commands

### DISCUSSION THREAD

**Discussion by jolau** [rxhanson/RectanglePro-Community#204] [General]
Is there something similiar to https://github.com/rxhanson/Rectangle/blob/master/TerminalCommands.md in Rectangle Pro? I need to adjust the snap edge margins, as they're way to small (would also be nice to have that in the settings).

**Comment by rxhanson** [comment]
Sorry for the delayed response here. You can use nearly all of the Terminal commands available for Rectangle, just replace `com.knollsoft.Rectangle` with `com.knollsoft.Hookshot`

I’ve actually given a lot of thought to adding a separate ui for all the hidden prefs and would like to add it at some point.

**Reply by panckreous** [reply]
a hidden cli that exposes everything to us who want it without muddying up the ui for normal people go together like... I dont know, ask chatgpt for a list of 10 things that go together well. But yes, mmmmm mm.
